var/spack/repos/builtin/packages/linux-external-modules/package.py

```diff
@@ -0,0 +1,338 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+from shutil import copy2
+
+
+class LinuxExternalModules(MakefilePackage):
+    """The Linux kernel provides services to start and stop programs,
+    handles the file system and other common "low-level" tasks that most
+    programs share, and schedules access to avoid conflicts when programs
+    try to access the same resource or device simultaneously. The kernel
+    has a modular design such that modules can be integrated as software
+    components. In this package, Linux has been configured to build out-of-tree
+    kernel modules."""
+
+    homepage = "https://github.com/torvalds/linux"
+    url = "https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.10.2.tar.xz"
+
+    maintainers("fleshling", "rountree", "rountree-alt")
```

Comment:
Is there a need for both `rountree` and `rountree-alt`?

Reply:
`rountree-alt` has been removed.